src/GuidesIndex.php

```diff
@@ -25,6 +25,24 @@ private static function getSummaries(
     return require(BuildPaths::GUIDES_SUMMARY);
   }
 
+  public static function search(string $term, SearchResultSet &$results): void {
```

Comment:
Here too. 
Also references are banned in strict mode. (Which this file sadly isn't :()